### ISSUE TITLE
data export

### ISSUE THREAD

**Issue by JanSchmidt3** [cantools/cantools#653]
Hi,
is there a way to export decoded frames to i.e. csv files? Maybe as extension of the plot command? 

I need to do some math with signals in different can frames and data from other sources. So, all data have different timestamps and I need to align them separately.

There are more than 300 signals and I need only about 10 which can perfectly matched by the plot-regex.

Jan

**Comment by andlaus** [comment]
I don't think that this is currently supported with the "included batteries" of cantools. That said it should be pretty easy to write a small script doing this...
